### PR TITLE
docs(choice-tile): implement and document vars and attrs

### PR DESCRIPTION
**Pull request by lalithkarikelli**
## Description

`hx-choicetile` :  implement and document variables and attributes

### What are the relevant story cards/tickets? Any additional PRs or other references?

Jira: SURF 2103

## Before you request a review for this PR:

- [x] For UI changes, did you manually test in recent versions of modern browsers (Chrome, Firefox, and Safari)?
- [x] For UI changes, did you manually test in IE11 and legacy Edge?
- [ ] Did you add component tests for any new code?
- [ ] Did you run the component unit tests via `yarn test` to ensure all tests passed?
- [ ] Did you include a screenshot of the component tests?
- [ ] If you changed/added functionality, did you update the demo page and documentation?
- [ ] If needed, did you add or modify the demo test page to test the changed/added functionality?
- [ ] Did you assign reviewers?
- [x] In Jira, have you linked to this PR on the ticket(s)?

### CODE REVIEW

File: src/scss/components/choice-tile/mixins/_hxChoiceTile.scss [hunk 1]
```diff
@@ -27,24 +25,22 @@
 @mixin hxChoiceTile--invalid($pseudo-state: null) {
   @if $pseudo-state == focus {
     box-shadow: $focus-glow-invalid;
-  }
-  @else if $pseudo-state == hover {
+  } @else if $pseudo-state == hover {
     background-color: $red-100;
```
Comment (BY00565233) [inline src/scss/components/choice-tile/mixins/_hxChoiceTile.scss:29]: Please write : background-color: var(--hxChoiceTile-invalid-hover-bgcolor, $red-100);

File: src/scss/components/choice-tile/mixins/_hxChoiceTile.scss [hunk 2]
```diff
@@ -27,24 +25,22 @@
...
     border-color: $red-900;
```
Comment (BY00565233) [inline src/scss/components/choice-tile/mixins/_hxChoiceTile.scss:30]: Please write : border-color : var(--hxChoiceTile-invalid-hover-bordercolor, $red-900);

File: src/scss/components/choice-tile/mixins/_hxChoiceTile.scss (hunk 1 above)
Comment (lalithkarikelli) [reply src/scss/components/choice-tile/mixins/_hxChoiceTile.scss]: Changes have been done, please review.

File: src/scss/components/choice-tile/mixins/_hxChoiceTile.scss (hunk 2 above)
Comment (lalithkarikelli) [reply src/scss/components/choice-tile/mixins/_hxChoiceTile.scss]: Changes have been done, please review.

**Review by BY00565233 — APPROVED** [review]
DEV LGTM

**Review by 100stacks — CHANGES_REQUESTED** [review]
### Please rebase and fix merge conflicts.

**Comment by lalithkarikelli** [comment]
> ### Please rebase and fix merge conflicts.

@100stacks fixed merge conflicts ,rebased with master, added new variables and squashed the commits

**Review by 100stacks — APPROVED** [review]
@lalithkarikelli, we'll work on small mods to variable naming at a future time.

Dev LGTM.

**Pull request merged**